### ISSUE TITLE
Parameters are not replaced in error message CWMMH0050E in french language

### ISSUE THREAD

**Issue by Kiiv** [OpenLiberty/open-liberty#27777]
**Describe the bug**  
The message log for error CWMMH0050E does not get its parameters replaced in French (at least) :

```
[2024-02-28T16:07:03.227+0100] 00000082 icroprofile.health31.services.impl.HealthCheck31ExecutorImpl W CWMMH0052W: La classe class io.openliberty.microprofile.health30.impl.HealthCheck30ResponseImpl qui implémente HealthCheckResponse dans lapplication {1} dans le module {2} a pour statut {3} avec les données suivantes : {4}.
```

**Steps to Reproduce**  
Implements a HealthCheck which failed.
Turn language in french by adding "-Duser.language=fr" in jvm.options configuration file.
Start the server and call the health check.

messages.log file trace a message with no paramters replaced

**Expected behavior**  
The message should contains the values of the paramters instead of {1}, {2}, {3} and {4}

**Diagnostic information:**  
 - OpenLiberty Version: 23.0.0.6
 - Affected feature(s) : mpHealth-3.1
 - Java Version:
 openjdk version "17.0.9" 2023-10-17
OpenJDK Runtime Environment (build 17.0.9+9-Ubuntu-122.04)
OpenJDK 64-Bit Server VM (build 17.0.9+9-Ubuntu-122.04, mixed mode, sharing)

**Additional context**  
Maybe it is something like some quotes need to be escaped because the quote as disapeared from "lapplication" compared with the original file https://github.com/OpenLiberty/open-liberty/blob/3ff6f11f3a68872e0af176fde71ec34d42fd706c/dev/io.openliberty.microprofile.health.internal.common/resources/io/openliberty/microprofile/health/resources/Health_fr.nlsprops

**Comment by pgunapal** [comment]
@Kiiv Thanks for bringing this to our attention. You're right, the issue seems to be related to how we format and parse the quotes in the messages that have replacement variables. We are working on a fix for this, I'll update this issue when the PR is merged.

**Comment by pgunapal** [comment]
The PR is merged, however, we would need to get the message files re-translated again with the correct quotes. I will circle back when the messages are translated and into a release.

**Comment by Kiiv** [comment]
Hi !

Do you have an ETA for this ? If I can help with the translation tell me.

**Comment by Kiiv** [comment]
For information I've also opened this one : https://github.com/OpenLiberty/open-liberty/issues/28048
If you think it should be an interesting addition, maybe it can be done together.

**Comment by pgunapal** [comment]
Hi @Kiiv, the translated messages are back, the fix should be included in the upcoming 24.0.0.4 fix pack.
Thanks!, I'll take a look at the new issue you opened.

**Comment by Kiiv** [comment]
Thank you very much for your support !
